examples/generate-fixtures.rs

```diff
@@ -0,0 +1,52 @@
+use radicle_avatar::{Avatar, Usage};
+use rand::{distributions::Alphanumeric, Rng};
+use serde::{Deserialize, Serialize};
+use std::{fs, iter};
+
+#[derive(Serialize, Deserialize)]
+struct Input {
+    input: String,
+    usage: Usage,
+}
+
+#[derive(Serialize, Deserialize)]
+struct TestCase {
+    input: Input,
+    output: Avatar,
+}
+
+fn main() -> std::io::Result<()> {
+    let mut test_cases: Vec<TestCase> = Vec::new();
+    let mut rng = rand::thread_rng();
+
+    for _ in 0..1000 {
+        let random_string: String = iter::repeat(())
+            .map(|()| rng.sample(Alphanumeric))
+            .map(char::from)
+            .take(40)
```

Comment:
We’re still restricting the set of input strings and not covering all inputs. We can fix this with
```suggestion
        let random_string_size = rng.gen_range(0..=40);
        let random_string: String = (0..random_string_size)
            .map(|_| rng.gen_range('\u{0000}'..char::MAX))
```

Reply:
This makes all of the fixtures unreadable:
```
  {
    "input": {
      "input": "񻕖򔂈􉲚󯹊򜀘񝪛񐌖𖔲񑡝쬁򋀜򤥻𪳮󻹧󵣎󷌩䌴𑎥􀈛򴖗縮񟩡򗥯򯖎񫜗󮤁򔉧",
      "usage": "any"
    },
    "output": {
      "emoji": "💡",
      "background": {
        "r": 168,
        "g": 135,
        "b": 110
      }
    }
  },
  {
    "input": {
      "input": "񻕖򔂈􉲚󯹊򜀘񝪛񐌖𖔲񑡝쬁򋀜򤥻𪳮󻹧󵣎󷌩䌴𑎥􀈛򴖗縮񟩡򗥯򯖎񫜗󮤁򔉧",
      "usage": "identity"
    },
    "output": {
      "emoji": "🤿",
      "background": {
        "r": 168,
        "g": 135,
        "b": 110
      }
    }
  },
```

Do we really want this?

Reply:
> Do we really want this?

I think that’s the price we’ll have to pay. Otherwise, the tests wouldn’t catch the issue I pointed out [here](https://github.com/radicle-dev/radicle-avatar/pull/5#discussion_r633626626).

Reply:
b555a04c24c723678bf0c30838a17d01c65b9533

Reply:
It's a bit weird that none of the inputs fall within the ascii range:
```
    "input": {
      "input": "򊹌𘗁򶘈򆝾󥞑󫦒񬹙􇏐񕘷𚤐󲸣󨌄󖲫򁷦𢞉򩲯䃫󦿩𰤝򳖮󑀥򿈖󝏤񈔱󵧨󄆙🪔򕲟𳨛󯔋񑐷󴠾򘩌󔦾󰻻񶄐𬝙󻠜",
    "input": {
      "input": "򊹌𘗁򶘈򆝾󥞑󫦒񬹙􇏐񕘷𚤐󲸣󨌄󖲫򁷦𢞉򩲯䃫󦿩𰤝򳖮󑀥򿈖󝏤񈔱󵧨󄆙🪔򕲟𳨛󯔋񑐷󴠾򘩌󔦾󰻻񶄐𬝙󻠜",
    "input": {
      "input": "򓽔񹘉񏴓񎯻󋇉ゝ𢬎󉥡󊞂񧕱􂨷􌈺𻂚򷒉",
    "input": {
      "input": "򓽔񹘉񏴓񎯻󋇉ゝ𢬎󉥡󊞂񧕱􂨷􌈺𻂚򷒉",
    "input": {
      "input": "򰥠냙򱔻񍑷𘝝􊣠򣧓􂐷𕙙󒵻􎆻񥱖񺭤󬹦󌌊򍦗񘐧󊯢󢮭𸊊񉹘𚟎􎱎𩠥񵨛󚼜󨩓򊻰󭬬򚄔򄐐򦨄򬠶񗉏񺗴􅽚񡦯򓦋􄗦",
    "input": {
      "input": "򰥠냙򱔻񍑷𘝝􊣠򣧓􂐷𕙙󒵻􎆻񥱖񺭤󬹦󌌊򍦗񘐧󊯢󢮭𸊊񉹘𚟎􎱎𩠥񵨛󚼜󨩓򊻰󭬬򚄔򄐐򦨄򬠶񗉏񺗴􅽚񡦯򓦋􄗦",
    "input": {
      "input": "򬱘⁗򼊚鸁򑷟􋩴𒚑󂑈򴌤",
    "input": {
      "input": "򬱘⁗򼊚鸁򑷟􋩴𒚑󂑈򴌤",
    "input": {
      "input": "񀒴󁦋󘇕򩎠񴺵󤴕",
    "input": {
      "input": "񀒴󁦋󘇕򩎠񴺵󤴕",
    "input": {
      "input": "񕓴򢨳󊮗񍞵𠺿𸟩􀜌򸐾𯃡򧃺茺",
    "input": {
      "input": "񕓴򢨳󊮗񍞵𠺿𸟩􀜌򸐾𯃡򧃺茺",
    "input": {
      "input": "񤫸犎󴕇󭃇𭞜񔌩􊎙󢾫𫼏󫟇񵰕򏫛󤦾󝴛𯫈򹐢񱠹򪑭򞐏򑌳򨅬򝘄𞏂𳶂直",
    "input": {
      "input": "񤫸犎󴕇󭃇𭞜񔌩􊎙󢾫𫼏󫟇񵰕򏫛󤦾󝴛𯫈򹐢񱠹򪑭򞐏򑌳򨅬򝘄𞏂𳶂直",
    "input": {
      "input": "񆦶񪡜",
    "input": {
      "input": "񆦶񪡜",
```

Reply:
> It's a bit weird that none of the inputs fall within the ascii range:

You could add some test cases that only use alphanumeric characters.

Reply:
We already have those as part of the regular tests.